### PR TITLE
set the default version to 1.0 (again)

### PR DESCRIPTION
**Pull request by MaximilianMeister**
this should have been done with:
https://github.com/crowbar/crowbar-client/pull/99

### CODE REVIEW

**Comment by rsalevsky** [comment]
+1

File: lib/crowbar/client/config.rb [hunk 1]
```diff
@@ -162,11 +162,9 @@ def default_anonymous
       #
       def default_apiversion
         if ENV["CROWBAR_APIVERSION"].present?
-          [
-            1.0, 2.0
-          ].include? ENV["CROWBAR_APIVERSION"]
+          ENV["CROWBAR_APIVERSION"].to_f
```
Comment (rhafer) [inline lib/crowbar/client/config.rb]: I know this has not been introduced here, but in general it's not a good idea to treat version number as floating point. Is there any good reason do to that here? (And elsewhere in the code)

**Comment by rhafer** [comment]
+1 apart from the float issue (which I guess is best handled in a separate PR)

File: lib/crowbar/client/config.rb (hunk 1 above)
Comment (MaximilianMeister) [inline lib/crowbar/client/config.rb]: I guess that was just for the looks (style of the code), as we did it in crowbar-init and crowbar-core as well. in the api constraint (where we check the version) it is converted to a string.

**Pull request merged**